LeadKit/LeadKit/Extensions/Support/UIScrollView+Support.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Copyright (c) 2017 Touch Instinct
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the Software), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED AS IS, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import UIKit
+
+public extension Support where Base: UIScrollView {
+
+    public var refreshControl: UIRefreshControl? {
+        if #available(iOS 10.0, *) {
+            return base.refreshControl
+        } else {
+            return base.subviews.first { $0 is UIRefreshControl } as? UIRefreshControl
+        }
+    }
+
+    public func setRefreshControl(_ newRefreshControl: UIRefreshControl?) {
+        if #available(iOS 10.0, *) {
+            base.refreshControl = newRefreshControl
+        } else {
+            if let control = newRefreshControl {
+                base.addSubview(control)
```

Comment:
tableView.refreshControl = refreshControl1
tableView.refreshControl = refreshControl2

This code add two refresh controls on tableView